activerecord/lib/active_record/tasks/database_tasks.rb

```diff
@@ -252,6 +259,19 @@ def load_seed
         end
       end
 
+      def self.run_cmd(cmd, args, action)
+        fail "Invalid database command - #{cmd}" unless valid_bin? cmd
```

Comment:
I figured it would be best not run to accept arbitrary commands. Too much paranoia?